### ISSUE TITLE
Use a CDN rather than keeping slick in the version control

### ISSUE THREAD

**Issue by Taitava** [Taitava/silverstripe-slickcarousel#11]
CDN links from the [slick website](kenwheeler.github.io/slick/#go-get-it):
```html
CSS

<link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css"/>

JS

<script type="text/javascript" src="//cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js"></script>
```

**NOTE that the version number above is different than the version of slick that we currently have in this repository.** I think it must be done so that the version number will be downgraded to what we are currently using. Then this can be released as a minor change, and maybe create another issue for inspecting whether it's worth to upgrade slick version by releasing a new major version of this module. It's not mandatory to release a new major version, because everyone who really needs a newer version of slick, can edit their YAML configuration file and change the slick version number (but only after this issue is solved!).

- [ ] Create the following configuration options:
 - slick_css_cdn_url
 - slick_js_cdn_url
 - slick_source (value can be either 'cdn' (default) or a string containing a path to a local slick installation directory)

- [ ] Add a readme note telling how to upgrade slick by editing the version number in the YAML config (`slick_css_cdn_url` and `slick_js_cdn_url` options).

- [ ] Remove the static slick files from this repository.

These are just very quick thoughts as I'm just quickly jotting down what came to my mind and didn't really look at all the changes that could possibly be needed to be done.

**Comment by Taitava** [comment]
The minor version upgrade will be available fro both 2.x and 1.x release lines (2.x: SS4 compatible, 1.x: SS3 compatible). However, if a major version change will be made **in addition** to a minor one, the major version update will only support SS4, and it will start a new release line 3.x (based on 2.x).
